examples/README.md

```diff
@@ -40,6 +40,19 @@ real applications with Kubernetes.
 Demonstrations of how to use specific Kubernetes features can be found in our [documents](../docs/).
 
 
+### Maintained Samples
+
+Name | Description
+------------- | ------------- | ------------
+[Guestbook](guestbook/) | PHP app with Redis
+[WordPress](mysql-wordpress-pd/) | WordPress with MySQL
```

Comment:
We should put better synopses here.

See:
https://github.com/openshift/origin/blob/master/examples/README.md